pkg/apis/autoscaling/v2beta2/conversion.go

```diff
@@ -48,7 +48,6 @@ func Convert_autoscaling_HorizontalPodAutoscaler_To_v2beta2_HorizontalPodAutosca
 	}
 	if behavior.ScaleUp != nil && behavior.ScaleUp.Tolerance != nil {
 		if !copiedAnnotations {
-			copiedAnnotations = true
```

Comment:
I want to keep this, even if we just have to ignore the lint... dropping this means that the variable is incorrect after this point if a later check / change was added

Reply:
Changed.